### PR TITLE
update mysql-wordpress example to use v1beta3 API

### PR DESCRIPTION
**Pull request by caesarxuchao**
@nikhiljindal 
update mysql-wordpress example to use v1beta3 API

### CODE REVIEW

File: examples/mysql-wordpress-pd/README.md [hunk 1]
```diff
@@ -130,28 +129,28 @@ If you want to do deeper troubleshooting, e.g. if it seems a container is not st
...
+metadata: 
+  labels: 
+    name: mysql
   name: mysql
+spec: 
+  ports:
+    # the port that this service should serve on
+    - port: 3306
+      # the port to access on the pods targeted by the service
+      targetPort: 3306
```
Comment (nikhiljindal) [inline examples/mysql-wordpress-pd/README.md]: targetPort is not required if it is same as port

File: examples/mysql-wordpress-pd/README.md [hunk 2]
```diff
@@ -227,26 +223,25 @@ Once the wordpress pod is running, start its service, specified by `wordpress-se
...
+    name: wpfrontend
   name: wpfrontend
-createExternalLoadBalancer: true
+spec: 
+  createExternalLoadBalancer: true
+  ports:
+    # the port that this service should serve on
+    - port: 80
+      # the port to access on the pods targeted by the service
+      targetPort: 80
```
Comment (nikhiljindal) [inline examples/mysql-wordpress-pd/README.md]: same

**Comment by nikhiljindal** [comment]
LGTM module a nit

**Comment by caesarxuchao** [comment]
Hi @nikhiljindal, I remove the "targetPort", could you take a look? Thanks.

**Comment by nikhiljindal** [comment]
LGTM, thanks!

**Pull request merged**